uiowa/uiowa
# Added validation for offset and items per page.

Resolves #7848 

# How to test

- Code review. Note: I removed our custom code to set offset in the block config because it gets set in the CTools block class we inherit from.
```
git checkout main && ddev blt ds --site sandbox.uiowa.edu && ddev drush @sandbox.local uli node/add/page
```
- Add a page.
- Add a list block to the layout and set it's items per page setting to ` ` (nothing). 
- Save the block **BUT DON'T SAVE THE LAYOUT.** Just leave the page open.
```
git checkout valid_items_per_page && ddev drush @sandbox.local updb
```
- Click "configure" on the block you created and check it's items per page. It should now be set to 50.
- Try putting in an empty value and confirm that you cannot save the form and it prints an error message (you may have to scroll back up).
- Trying putting in a numeric value outside the 1-50 range and confirm that you cannot save the form and it prints an error message.
- Trying putting in an empty value for the "offset" field and confirm that you cannot save the form.
